### ISSUE TITLE
battery always 100%

### ISSUE THREAD

**Issue by Robert-Riedl** [Drolla/WavePlus_Bridge#23]
I've just realised that the script always reads the battery at 100%

```

connect to 2930144663
Initialize scanning/discovery
  Run scan
  Found device None
  Found device 2930144663
  Device 2930144663 found, MAC address=None
Control characteristics: Handle=16/16
  CCCD value (indication disabled): b'\x00\x00'
  CCCD value (indication enabled): b'\x02\x00'
Waiting on notificaiton
  Received notification (16): b'm\x00G^\x01\x00\x02O\x00\x82\x04 \x00\x00\x00\x00@L\x14\x00'
  Received notification (16): b'\x8e3\x08\x00p\x00L\x0b\t\x00'
Received data: b'm\x00G^\x01\x00\x02O\x00\x82\x04 \x00\x00\x00\x00@L\x14\x00\x8e3\x08\x00p\x00L\x0b\t\x00'
  -> s={'humidity': 44.0, 'radon_st': 45, 'radon_lt': 45, 'temperature': 21.09, 'pressure': 993.92, 'co2': 1151.0, 'voc': 97.0}, c={'illuminance': 79, 'battery': 100}
+------------+------------+------------+------------+------------+------------+------------+------------+------------+------------+
|  2930144663|        44.0|          45|          45|       21.09|      993.92|      1151.0|        97.0|          79|         100|
+------------+------------+------------+------------+------------+------------+------------+------------+------------+------------+
Reading sensor data for device 2930144663
Connect to 2930144663
Control data: Address=f4:60:77:6f:a9:fb, used cached data: b'm\x00G^\x01\x00\x02O\x00\x82\x04 \x00\x00\x00\x00@L\x14\x00\x8e3\x08\x00p\x00L\x0b\t\x00'
  -> s={'humidity': 44.0, 'radon_st': 45, 'radon_lt': 45, 'temperature': 21.09, 'pressure': 993.92, 'co2': 1151.0, 'voc': 97.0}, c={'illuminance': 79, 'battery': 100}
+------------+------------+------------+------------+------------+------------+------------+------------+------------+------------+
|  2930144663|        44.0|          45|          45|       21.09|      993.92|      1151.0|        97.0|          79|         100|
+------------+------------+------------+------------+------------+------------+------------+------------+------------+------------+
Reading sensor data for device 2930144663
Connect to 2930144663
Control data: Address=f4:60:77:6f:a9:fb, used cached data: b'm\x00G^\x01\x00\x02O\x00\x82\x04 \x00\x00\x00\x00@L\x14\x00\x8e3\x08\x00p\x00L\x0b\t\x00'
  -> s={'humidity': 44.0, 'radon_st': 45, 'radon_lt': 45, 'temperature': 21.09, 'pressure': 993.92, 'co2': 1151.0, 'voc': 97.0}, c={'illuminance': 79, 'battery': 100}
+------------+------------+------------+------------+------------+------------+------------+------------+------------+------------+
|  2930144663|        44.0|          45|          45|       21.09|      993.92|      1151.0|        97.0|          79|         100|
+------------+------------+------------+------------+------------+------------+------------+------------+------------+------------+


```

The Android App shows 89%.

**Comment by Drolla** [comment]
Thanks for reporting this issue. Strangely I see the same thing on my side now. I will look into it.

**Comment by Drolla** [comment]
This bug has been fixed with commit 142bea34a2d08df361e88bc2de6c77fdee7f1f75.

**Comment by Robert-Riedl** [comment]
can confirm, it works - thanks
